initializr-service/src/main/resources/application.yml

```diff
@@ -22,6 +22,14 @@ spring:
 initializr:
   env:
     boms:
+      spring-cloud-gcp:
+        groupId: org.springframework.cloud
+        artifactId: spring-cloud-gcp-dependencies
+        versionProperty: spring-cloud-gcp.version
+        mappings:
+          - versionRange: "1.0.0.M2"
```

Comment:
`versionRange` is the version of Spring Boot (check other entries). I guess if you  require `2.0.0.RC1` and later, it should be `2.0.0.RC1`